base.cfg

```diff
@@ -85,7 +85,7 @@ access-log-args  = ("${buildout:directory}/var/log/${:_buildout_section_name_}-Z
 access-log-kwargs = {"when": "D", "interval": 1, "backupCount": 14}
 
 [euphorie.ini]
-recipe = collective.recipe.template
+recipe = collective.recipe.template[genshi]
 client = http://client-oirademo/
 smartprintng_url = ${settings:smartprintng-url}
 library = ${settings:library}
```

Comment:
This part is not needed anymore since quite some time, right?
I would just remove it.

Reply:
Wouldn't we lose the ability to run alembic upgrades via the alembic cli?

Reply:
Has this something to do with the alembic cli?
I think this was used as a sort of external registry via z3c.appconfig.
See https://github.com/euphorie/Euphorie#compatibility

Reply:
Ah, sorry, I mixed something up.